make test fails when GOPATH contains multiple paths

Running `make test` when GOPATH contains multiple paths outputs an error message saying that moq tool is not found, although it is in fact installed.

```
$ echo $GOPATH
/Users/lucas.mundim/Projects/golang:/Users/lucas.mundim/.gvm/pkgsets/go1.10.8/global

$ ls /Users/lucas.mundim/.gvm/pkgsets/go1.10.8/global/bin/
moq

$ ls /Users/lucas.mundim/Projects/golang/bin/
moq

$ make test
Building for GOARCH=amd64
Everything is gofmt approved!
ERROR: 'moq' tool is needed to update mock test files, install it with:
go get github.com/matryer/moq

make: *** [pkg/controllers/proxy/network_services_controller_moq.go] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -155,7 +155,7 @@ ifeq "$(BUILD_IN_DOCKER)" "true"
 	$(DOCKER) run -v $(PWD):/go/src/github.com/cloudnativelabs/kube-router -w /go/src/github.com/cloudnativelabs/kube-router $(DOCKER_BUILD_IMAGE) \
 			sh -c 'apk add --no-cache git build-base && go get github.com/matryer/moq && go generate -v $(*).go'
 else
-	@test -x $(GOPATH)/bin/moq && exit 0; echo "ERROR: 'moq' tool is needed to update mock test files, install it with: \ngo get github.com/matryer/moq\n"; exit 1
+	@test -x $(lastword $(subst :, ,$(GOPATH)))/bin/moq && exit 0; echo "ERROR: 'moq' tool is needed to update mock test files, install it with: \ngo get github.com/matryer/moq\n"; exit 1
 	go generate -v $(*).go
 endif
 

